src/pool/srv_pool.c

```diff
@@ -1151,11 +1187,17 @@ pool_svc_alloc_cb(d_iov_t *id, struct ds_rsvc **rsvc)
 	svc->ps_svc_rf = -1;
 	svc->ps_force_notify = false;
 
+	rc = ds_pool_lookup(svc->ps_uuid, &svc->ps_pool);
+	if (rc != 0) {
+		D_INFO(DF_UUID ": look up pool: " DF_RC "\n", DP_UUID(svc->ps_uuid), DP_RC(rc));
```

Comment:
very minor, I suppose this could be DL_INFO(rc, ...)?

Reply:
@kccain, sure, I'll find a lost-cost chance to change this into a `DL_INFO`.